features/support/env.rb

```diff
@@ -124,7 +129,8 @@
 # This would set `behavior = :raise`, but that wasn't added until Rails 4.
 ActiveSupport::Deprecation.behavior = -> message, callstack do
   e = StandardError.new message
-  e.set_backtrace callstack
+  e.set_backtrace callstack.map(&:to_s)
+  puts e
```

Comment:
I documented the reason for it in this commit: 169266636363278c903dc87965f6501778c2fa05

> & call puts for deprecation in Cucumber, because they sometimes won’t
be printed and you’ll be left with a failed test but no failure message